state/executor.go

```diff
@@ -87,12 +87,12 @@ type BlockResult struct {
 	TotalGas uint64
 }
 
-// ProcessBlock already does all the handling of the whole process
+// ProcessBlock already does all the handling of the whole process, TODO
```

Comment:
Why we need TODO here. Is it leftover or something is missing?

Reply:
Removed at a75fef74550531c07c5d423278a50f9968708952